app/views/WishList/index.tsx

```diff
@@ -0,0 +1,294 @@
+                        variant="secondary"
+                        icons={<IoTrash />}
+                    >
+                        Remove
+                    </Button>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+function WishList() {
+    const [page, setPage] = useState<number>(1);
+    const [pageSize, setPageSize] = useState<number>(10);
+    const alert = useAlert();
+    const [
+        deleteWishlist,
+    ] = useMutation<RemoveWishListMutation, RemoveWishListMutationVariables>(
+        REMOVE_WISH_LIST,
+        {
+            onCompleted: (response) => {
+                refetch();
+                if (response?.ok) {
+                    alert.show(
+                        'Item deleted from wishlist successfully',
+                        {
+                            variant: 'success',
+                        },
+                    );
+                } else {
+                    alert.show(
+                        'Failed to delete item from wishlist',
+                        {
+                            variant: 'error',
+                        },
+                    );
+                }
+            },
+            onError: () => {
+                alert.show(
+                    'Failed to delete item from wishlist',
+                    {
+                        variant: 'error',
+                    },
+                );
+            },
+        },
+    );
+
+    const [
+        createCartItem,
+    ] = useMutation<CreateCartMutation, CreateCartMutationVariables>(
+        CREATE_CART,
+        {
+            onCompleted: () => refetch(),
+        },
+    );
+
+    const {
+        data,
+        refetch,
+        loading,
+    } = useQuery<WishListQuery, WishListQueryVariables>(
+        WISH_LIST, {
+            variables: {
+                page,
+                pageSize,
+            },
+            onCompleted: (res: WishListQuery) => {
+                setPage(res.wishList?.page ? res.wishList.page : page);
+                setPageSize(res.wishList?.pageSize ? res.wishList.pageSize : pageSize);
+            },
+        });
+
+    const addToCart = useCallback((id: string, quantity: number) => {
+        createCartItem({
+            variables: {
+                id,
+                quantity,
+            },
+        });
+    }, []);
+
+    const deleteBook = useCallback((id: string) => {
+        deleteWishlist({
+            variables: {
+                id,
+            },
+        });
+    }, []);
+
+    const wishes = (data?.wishList?.results) ? data.wishList.results : [];
+    const wishItemRendererParams = React.useCallback((_, d: Wish) => ({
+        wish: d,
+        onRemoveWishList: deleteBook,
+        onCreateCart: addToCart,
+    }), []);
+
+    return (
+        <div className={styles.wishList}>
+            <Container
+                className={styles.featuredBooksSection}
+                heading="My Wishlist"
```

Comment:
Inconsistent name for "Wishlist".
Let's make sure we are using either "Wish list" or "Wishlist"